src/routes/v1/index.js

```diff
@@ -1,8 +1,10 @@
 import authRoutes from './auth.route';
 import userRoutes from './user.route';
+import ratingArticleRoutes from './articlesRatings.routes';
 
 export default app => {
   app.use('/auth', authRoutes);
   app.use('/api/v1/users', userRoutes);
   app.use('/api/v1/auth', userRoutes);
+  app.use('/api/v1', ratingArticleRoutes);
```

Comment:
I think there could be a better way to implement this route. You could leave it on the `users` route since it is tied to the user

Reply:
why are you mounting the ratings controller on the base route. Follow the example from others. As a matter if fact, rating article should be a method on the `ArticleController`. Therefore the route should be in the article routes. Communicate with @tobechukwuobitube to handle that

Reply:
In that case, I think it should be `articles`